imports/wasi_snapshot_preview1/testdata/cargo-wasi/wasi.rs

```diff
@@ -0,0 +1,39 @@
+use std::env;
+use std::fs;
+use std::io;
+use std::io::Write;
+use std::process::exit;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    match args[1].as_str() {
```

Comment:
I merged these as otherwise our repo size will grow a lot as we test more. we do something similar in gojs where we compile a runner wasm once